src/client/Microsoft.Identity.Client/AppConfig/ExternalTokenResult.cs

```diff
@@ -0,0 +1,49 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading;
+using System.Threading.Tasks;
+
+namespace Microsoft.Identity.Client
+{
+    /// <summary>
+    /// Token result from external token provider
+    /// </summary>
+    public class ExternalTokenResult
+    {
+        /// <summary>
+        /// Correlationn Id to track request
+        /// </summary>
+        public string CorrelationId { get; set; }
```

Comment:
I think we should remove this. I don't see the point of overrideing the correlation id. We should just use the correlation id from the input.

Reply:
Yeah, it was in the spec so I put it here, will remove it.